python_modules/libraries/dagster-azure/dagster_azure_tests/adls2_tests/conftest.py

```diff
@@ -5,7 +5,7 @@
 
 @pytest.fixture(scope="session")
 def storage_account():
-    yield "elementldevstorage"
+    yield "dagsterdev"
```

Comment:
with a new azure cloud setup comes a new storage account name and access key - how do i update buildkite to have the right credentials?

Reply:
i can follow up offline